subsys/bluetooth/mesh/pb_gatt_srv.c

```diff
@@ -38,6 +38,12 @@ LOG_MODULE_REGISTER(bt_mesh_pb_gatt_srv);
 
 #define FAST_ADV_TIME (60LL * MSEC_PER_SEC)
 
+#define BT_DEVICE_NAME (IS_ENABLED(CONFIG_BT_DEVICE_NAME_DYNAMIC) ? \
```

Comment:
I'm not big fun of copy-paste approach but didn't find correct header for this definition :(

Reply:
Created a separate header file. Also moved macros for adv intervals as they are used by both pb_gatt_srv.c and proxy_srv.c.